src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesInstaller.Codeunit.al

```diff
@@ -34,4 +35,27 @@ codeunit 329 "No. Series Installer"
 
         UpgradeTag.SetUpgradeTag(NoSeriesUpgradeTags.GetImplementationUpgradeTag());
     end;
+
+    local procedure TriggerMovedTableSchemaCheck()
```

Comment:
The name should be updated that this code is using OnInstall trigger and explain more about the usage of this method.

Reply:
it's a local in an install codeunit... how is that not clear enough?

Reply:
added to upgrade too